packages/card/src/ux-card.ts

```diff
@@ -51,13 +51,17 @@ export class UxCard implements UxComponent {
     for (let i = 0; i < 10; i++) {
       this.element.classList.remove(`ux-card--${size}-${i}`);
       this.element.classList.remove(`ux-card--order-${this.order}-${size}-${i}`);
+      this.element.classList.remove(`ux-grid-cell--${size}-${i}`);
+      this.element.classList.remove(`ux-grid-cell--order-${this.order}-${size}-${i}`);
```

Comment:
the api `.add`, `.remove` of class list can take a list of argument. so this can be changed to:
```suggestion
      this.element.classList.remove(
        `ux-card--${size}-${i}`,
        `ux-card--order-${this.order}-${size}-${i}`,
        `ux-grid-cell--${size}-${i}`,
        `ux-grid-cell--order-${this.order}-${size}-${i}`
      );
```
Same for `add`